### PR TITLE
MONIT-29710: created new version for query suggestions files

### PR DESCRIPTION
**Pull request by serinekar**

### CODE REVIEW

File: query_suggestions_v2/max_raw_tip.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+**Option 2:** if you don’t need to interpolate the underlying data,
```
Comment (Margarita-Staneva) [inline query_suggestions_v2/max_raw_tip.md:1]: If?

File: query_suggestions_v2/min_more.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Standard aggregation functions like [[queryFunctionName type=header text=min()]] interpolate data in each time series. The aggregation function  is applied to the interpolated series.  Aggregating thousands of time series affects query performance.
```
Comment (Margarita-Staneva) [inline query_suggestions_v2/min_more.md:1]: I see double spaces between function and is and after the full stop.

The aggregation function  is applied to the interpolated series.  Aggregating thousands of time series affects query performance.

File: query_suggestions_v2/min_raw_tip.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+
+**Option 2:** if you don’t need to interpolate the underlying data,
```
Comment (Margarita-Staneva) [inline query_suggestions_v2/min_raw_tip.md:2]: If should start with a capital letter.

File: query_suggestions_v2/percentile_raw_tip.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+
+**Option 2:** if you don’t need to interpolate the underlying data,
```
Comment (Margarita-Staneva) [inline query_suggestions_v2/percentile_raw_tip.md:2]: If?

File: query_suggestions_v2/sum_more.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+Standard aggregation functions like  [[queryFunctionName type=header text=sum()]] interpolate data in each time series. The aggregation function is applied to the interpolated series. Aggregating thousands of time series affects query performance.
```
Comment (Margarita-Staneva) [inline query_suggestions_v2/sum_more.md:1]: Do we need the double spaces between like  and [[?

**Review by Margarita-Staneva — COMMENTED** [review]
I've got some minor comments, rest looks good.

**Comment by serinekar** [comment]
> I've got some minor comments, rest looks good.

@Margarita-Staneva I have fixed your comments. Thanks for catching them.

**Review by Margarita-Staneva — APPROVED** [review]
Looks good to me

**Pull request merged**